RabbitMQ/mini_lab/producer/publish.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2024 Bloomberg Finance L.P.
```

Comment:
We need to update the file. right now it doesn't have execution permissions when attempting to run.